### PR TITLE
compute dynamical time

### PR DESCRIPTION
**Pull request by Yurlungur**
## PR Summary

Script to compute the dynamical time of a tracer particle while it's nucleosynthetically active. Done in time parallel, so per-tracer data is not needed.

## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

**Comment by Yurlungur** [comment]
Details:
1. Nucleosynthetically active means temperatures between 1 and 10 GK
2. Dynamical time is defined as rho/rhodot

Algorithm is time parallel and is as follows:
1. Generate an "integration array" of total amount of time that is nucleosynthetically active and total sum of rho/rhodot
2. For even-numbered tracer dumps, load dump at i and i-1. Compute dt, drhodt, rho/drhodt (rho is time-centered. It's the average between dump at i and dump at i-1)
3. For each tracer if 1 GK < T < 10 GK, add dt and dt*(rho/drhodt) to accumulation arrays. Otherwise add zero.
4. Average at the end.

File: script/analysis/get_dynamical_time_time_parallel.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    dt_int = np.zeros_like(tau_dyns_int)
+    
+    for dump_id in dump_ids:
+        print("...",dump_id)
+        fnam_now = generate_filename(parent_dir, dump_id)
+        fnam_last = generate_filename(parent_dir, dump_id - 1)
+        td_now = get_td_from_fnam(fnam_now)
+        td_last = get_td_from_fnam(fnam_last)
+        
+        dt = td_now['time'][0] - td_last['time'][0]
```
Comment (brryan) [inline script/analysis/get_dynamical_time_time_parallel.py:74]: Should this `dt` be proper time for the tracer particle rather than coordinate time?

Comment (Yurlungur) [reply script/analysis/get_dynamical_time_time_parallel.py:74]: Uh hmm... I'm not sure.

Comment (Yurlungur) [reply script/analysis/get_dynamical_time_time_parallel.py:74]: Yes probably but it probably doesn't matter too much.

**Comment by Yurlungur** [comment]
@brryan other than the proper time, do you see any issues? If not do you want to approve?

**Review by brryan — APPROVED** [review]

**Pull request merged**